packages/react-google-maps-api/src/components/places/StandaloneSearchBox.tsx

```diff
@@ -42,12 +49,7 @@ class StandaloneSearchBox extends PureComponent {
   componentDidMount = () => {
     const searchBox = new google.maps.places.SearchBox(
       this.containerElement.current.querySelector('input'),
-      Object.assign(
-        {
-          map: this.context
-        },
-        this.props.options
-      )
```

Comment:
)))))

Reply:
Search box doesn't accept map in constructor or in options